web-src/src/components/PromptEditor.js

```diff
@@ -196,6 +229,22 @@ function PromptEditor({ isOpen, onClose, ...props }) {
               readOnly={!viewSource}
             />
           </div>
+
+          <Flex gap="size-100" UNSAFE_className={editorSyntaxError ? style.hidden : style.validHelpText }>
+            <CheckmarkCircle aria-label="Positive Alert" color="positive" />
+            <Text>
+              Prompt is valid
+            </Text>
+          </Flex>
+
+          <Flex gap="size-100" UNSAFE_className={editorSyntaxError ? style.errorHelpText : style.hidden}>
+            <Alert aria-label="Negative Alert" color="negative" />
+            <Text>
+              The characters <span style={{ fontWeight: '600' }}>&#123;</span>, <span style={{ fontWeight: '600' }}>&#125;</span>,
+              and <span style={{ fontWeight: '600' }}>&quot;</span> are reserved and can&apos;t be used within quoted text values.
+              Please remove or replace these characters and try again.
+            </Text>
```

Comment:
Localizable text. It's a good opportunity to localize a very complex text string.
Please refer to this example: https://github.com/adobe/aem-genai-assistant/blob/3c26d313fbea58515e8710e209144592c0a99356/web-src/src/components/ConsentDialog.js#L72